build.gradle

```diff
@@ -112,26 +125,35 @@ spotless {
 check.dependsOn 'spotlessApply'
 
 allprojects {
-    apply plugin: 'base'
+    task copyDockerTest(type: Copy) {
+        delete "build/docker"
 
-    afterEvaluate { project ->
-        def composeDeps = [
-                ":airbyte-config:init",
-                ":airbyte-db:lib",
-                ":airbyte-migration",
-                ":airbyte-scheduler:app",
-                ":airbyte-workers",
-                ":airbyte-server",
-                ":airbyte-webapp",
-        ].toSet().asImmutable()
-
-        if (project.getPath() in composeDeps) {
-            composeBuild.dependsOn(project.getPath() + ':assemble')
-        }
+        from "${project.projectDir}/Dockerfile"
+        into "build/docker/"
+    }
+
+    // Use task types
+    task buildDockerImageTest(type: DockerBuildImage) {
```

Comment:
These tasks are also used for generating the images that we'll publish, so we should be able to drop the test.

Reply:
Done, in this specific commits it was a test for factorizing the task. Now that it is generalized I have remove the test suffix. 

I should have put it back in draft, sorry about that. I usually re-request a review after I finish all I wanted to push (In this case I pushed to github a temporary commit as a backup solution, just in case anything happen to my laptop).